### ISSUE TITLE
Import script doesn't work on Windows

### ISSUE THREAD

**Issue by hhyyrylainen** [Revolutionary-Games/Thrive#802]
Likely due to command line argument parsing being broken there.
Once done the artist workflow page on the wiki needs to be updated.

- [x] script works
- [x] importing tutorial is updated
- [x] a build is available for artists to use

**Comment by ShimmyShaman** [comment]
which script is it?
LeviathanImporterCache?

> importing tutorial is updated
 where's the non-updated tutorial?

Is this intended to use via command line arguments when starting Thrive.exe; open thrive from a console window (the C-make option set(CREATE_CONSOLE_APP ON)), or open a console from the initialised engine (ConsoleInput::CreateConsoleWindow()), and import and update the models while the game is going on?

**Comment by hhyyrylainen** [comment]
The script is called `AssetImportHelper.rb` and is generated by `MakeRelease.rb`.

I just fixed the importing for windows (it was bad command line parameter handling):
https://github.com/hhyyrylainen/Leviathan/commit/20953919d94c1a95c84596f422b45583cf67a19b

Right now it is just a command line tool (LeviathanEditor), but in the future it would be nice to have a GUI for it and live previewing.

The importing tutorial is also updated: https://wiki.revolutionarygamesstudio.com/wiki/How_to_Import_Assets
and I made a build for the artists to use. I'm just waiting a bit before closing this to see if the artists manage to use the tool.
